lessons/4_numbers/index.html

```diff
@@ -354,6 +347,28 @@ <h3>Animal List Example Code</h3>
                     </code>
                 </pre>
             </section>
+            <section>
+                <h3>Creating elements with JS!</h3>
+                <pre>
+                    <code>
+// HTML: <p id="one">Travis</p>
+// same as
+
+const nameEl = document.createElement('p');
+nameEl.textContent = "Travis";
+
+document.body.appendChild(nextEl);
+                    </code>
+                </pre>
+                <ul>
+                    <li><code>document.createElement</code> is a function that create dom elements</li>
+                    <li><code>el.appendChild</code> is a function that adds elements to the dom</li>
+                </ul>
+            </section>
+            <section>
+                <h3>Class work</h3>
+                <p>Time to break into small groups and make this:</p>
+                <img src="./lesson_4_demo.png" />
```

Comment:
I would say that just picture is not enough. I'm myself can imagine what needs to be done, but students would not. It's better that they have something what they can read as well - like explanation what exactly needs to be done.

Reply:
Yeah I'm thinking separate markdown file like we do for homework so it is easy to reference. Also in the process of actually making what is in the photo

Reply:
Added classwork markdown file with is a spec of the mini app to be worked on.